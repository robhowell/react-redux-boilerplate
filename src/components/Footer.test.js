import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Components/Footer Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
