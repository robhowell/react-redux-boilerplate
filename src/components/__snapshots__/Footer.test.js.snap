// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Footer Component should render correctly 1`] = `
<footer>
  <p>
    Created with
     
    <span
      aria-label="love"
      role="img"
    >
      ❤️
    </span>
     
    by 
    <a
      href="https://github.com/tomglenn"
    >
      Tom Glenn
    </a>
  </p>
</footer>
`;
